Sources/ClientRuntime/Retries/RetryStrategy.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public protocol RetryStrategy {
```

Comment:
This protocol is based on one of the same name in reference architecture.

There is an implementation of this protocol above called `DefaultRetryStrategy`.